cmake/modules/FindPThread.cmake

```diff
@@ -14,19 +14,36 @@
 
 SET(PTHREAD_FIND_REQUIRED ${PThread_FIND_REQUIRED})
 
-find_path(PTHREAD_INCLUDE_DIR pthread.h)
-mark_as_advanced(PTHREAD_INCLUDE_DIR)
-
-set(PTHREAD_NAMES ${PTHREAD_NAMES} pthread libpthread)
-find_library(PTHREAD_LIBRARY NAMES ${PTHREAD_NAMES} )
-mark_as_advanced(PTHREAD_LIBRARY)
-
-# handle the QUIETLY and REQUIRED arguments and set PTHREAD_FOUND to TRUE if
-# all listed variables are TRUE
-include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(PThread DEFAULT_MSG PTHREAD_LIBRARY PTHREAD_INCLUDE_DIR)
-
-IF(PTHREAD_FOUND)
-  SET(PThread_LIBRARIES ${PTHREAD_LIBRARY})
-  SET(PThread_INCLUDE_DIRS ${PTHREAD_INCLUDE_DIR})
-ENDIF(PTHREAD_FOUND)
+if(WIN32)
```

Comment:
Do check whether `https://cmake.org/cmake/help/v3.0/module/FindThreads.html` just works.
I wanted to switch to that from this module, but did not.